libs/components/pages/src/lib/modules/action-hub/fixtures/action-hub-fixture.module.ts

```diff
@@ -0,0 +1,23 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { SkyKeyInfoModule } from '@skyux/indicators';
+
+import { SkyActionHubModule } from '../action-hub.module';
+
+import { ActionHubAsyncFixtureComponent } from './action-hub-async-fixture.component';
+import { ActionHubContentFixtureComponent } from './action-hub-content-fixture.component';
+import { ActionHubInputsFixtureComponent } from './action-hub-inputs-fixture.component';
+import { ActionHubRecentSvcFixtureComponent } from './action-hub-recent-svc-fixture.component';
+import { ActionHubSyncFixtureComponent } from './action-hub-sync-fixture.component';
+
+@NgModule({
+  imports: [CommonModule, SkyActionHubModule, SkyKeyInfoModule],
+  declarations: [
+    ActionHubAsyncFixtureComponent,
+    ActionHubContentFixtureComponent,
+    ActionHubInputsFixtureComponent,
+    ActionHubRecentSvcFixtureComponent,
+    ActionHubSyncFixtureComponent,
+  ],
+})
+export class SkyActionHubFixtureModule {}
```

Comment:
I created this fixture module and declared all the components in it, then imported it into `TestBed`. otherwise there were red squigglies everywhere in the IDE since Angular Language Services couldn't determine the module where each component was declared.